app/(main)/generate/TabStep.tsx

```diff
@@ -15,9 +15,10 @@ export default function TabStep({ stepNumber, hidden, text }: Props) {
     <TabsTrigger
       value={`step-${stepNumber}`}
       className={`
-       grid
+      grid
        min-h-[70px]
-      grow-[1]  grid-flow-row rounded-none bg-[#EEF0FD]
+       grow-[1]
+      grid-flow-row  rounded-none bg-[#EEF0FD] bg-[#F9F9F9]
```

Comment:
You have an extra whitespace between `grid-flow-row rounded-none`


You also have two `bg-[]` CSS rule. Pick one over the other. Not both.